django/core/management/utils.py

```diff
@@ -106,3 +106,14 @@ def parse_apps_and_model_labels(labels):
             apps.add(app_config)
 
     return models, apps
+
+
+def preparse_commandline_option(argv, option):
+        parser = CommandParser(add_help=False, allow_abbrev=False)
+        parser.add_argument(option, dest='value')
+
```

Comment:
Please add a docstring and remove this blank line. Also, this block is indent 8 spaces instead of 4.